### PR TITLE
Fix 3918 change datetime hourcycle

### PR DESCRIPTION
**Pull request by kumarav1nash**
## What does this PR do?

This PR fixes the DateTime issue which is related to #3918

## Test Plan

This was a UI issue so took a screenshot of it after fixes.
Screenshot 2022-10-23 at 4 02 41 PM

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: appwrite/.env [hunk 1]
```diff
@@ -0,0 +1,90 @@
+_APP_ENV=production
```
Comment (stnguyen90) [inline appwrite/.env:1]: Please remove this file

File: appwrite/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,703 @@
+x-logging: &x-logging
```
Comment (stnguyen90) [inline appwrite/docker-compose.yml:1]: Please remove this file

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 please remove the extra files added to the repo.

File: appwrite/.env (hunk 1 above)
Comment (kumarav1nash) [reply appwrite/.env]: Done

**Comment by kumarav1nash** [comment]
sure

**Review by stnguyen90 — APPROVED** [review]
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

**Comment by stnguyen90** [comment]
Sorry, it looks like this isn't applicable anymore with the release of 1.1.0

**Pull request closed without merging**